GitCommands/CommitDataManager.cs

```diff
@@ -146,35 +148,32 @@ private bool TryGetCommitLog(string commitId, string format, [NotNullWhen(return
             return true;
         }
 
-        private static string ProcessDiffNotes(int startIndex, string[] lines)
+        private static string ProcessDiffNotes(int startIndex, string[] lines, out string? notes)
```

Comment:
We should return the tuple here instead of the `out` param
```suggestion
        private static (string RawBody, string? RawNotes) ProcessDiffNotes(int startIndex, string[] lines)
```